public class Main {
    public static void main(String[] args) {

        // ===== UC5 =====
        if (args.length == 0) {
            System.out.println("Hello, World!");
        } else {
            System.out.print("Hello ");

            for (String name : args) {
                System.out.print(name + " ");
            }
        }
    }
}